### PR TITLE
Null safe check if `isBackgroundPlaybackEnabled` property is set

### PR DESCRIPTION
**Pull request by strangesource**
## Description
`isBackgroundPlaybackEnabled` was checked in an unsafe way which caused attaching a player to a view to crash if  `isBackgroundPlaybackEnabled` was not set.

## Changes
- make `isBackgroundPlaybackEnabled` check null safe

## Checklist
- [x] 🗒 `CHANGELOG` entry

### CODE REVIEW

**Comment by strangesource** [comment]
@123mpozzi is it fine that the property is not set in this scenario or do we have to have a closer look why it is unset?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [Unreleased]
+
+### Fixed
+
+- Error where setting `PlaybackConfig.isAutoplayEnabled = true` causes the player view creation to fail on Android
+
```
Comment (123mpozzi) [inline CHANGELOG.md:8]: Nit -- Wording: isn't the crash happening when the flag is not set?

```suggestion
- Error where not setting `PlaybackConfig.isAutoplayEnabled` causes the player view creation to fail on Android

```

**Review by 123mpozzi — APPROVED** [review]
Nice catch!! 💪 
The flag should absolutely be null safe

File: CHANGELOG.md (hunk 1 above)
Comment (strangesource) [reply CHANGELOG.md:8]: No, the crash happens on the `isBackgroundPlaybackEnabled` flag if `isAutoplayEnabled` is set to true.

Comment (123mpozzi) [reply CHANGELOG.md:8]: Ah I see, thanks for clarifying

**Pull request merged**